app/src/main/java/beepbeep/pixelsforredditx/common/HeaderFooterPagedListAdapter.kt

```diff
@@ -9,7 +9,7 @@ abstract class HeaderFooterPagedListAdapter<T>(diffCallback: DiffUtil.ItemCallba
     open var hasHeader: Boolean = false
     open var hasFooter: Boolean = false
 
-    fun HeaderFooterAdapter(hasHeader: Boolean, hasFooter: Boolean) {
+    fun headerFooterAdapter(hasHeader: Boolean, hasFooter: Boolean) {
```

Comment:
I think this file can actually be removed

Reply:
I will remove it then!